charts/gatekeeper/templates/gatekeeper-controller-manager-deployment.yaml

```diff
@@ -58,21 +58,21 @@ spec:
         - --enable-external-data={{ .Values.enableExternalData }}
         - --log-mutations={{ .Values.logMutations }}
         - --mutation-annotations={{ .Values.mutationAnnotations }}
-        
+
```

Comment:
This whitespace change should be reflected in `manifest_staging/chart` first instead of `chart/`

Reply:
I do not understand. How do I get rid of those?